app/Http/Controllers/NewsController.php

```diff
@@ -79,4 +82,27 @@ public function update($id)
 
         return ['message' => trans('news.update.ok')];
     }
+
+    private function sidebarMeta($post)
+    {
+        if ($post !== null && $post->published_at !== null) {
+            $currentYear = $post->published_at->year;
+        }
+
+        $currentYear = $currentYear ?? date('Y');
+
+        $years = NewsPost::selectRaw('DISTINCT YEAR(published_at) year')
+            ->whereNotNull('published_at')
+            ->orderBy('year', 'DESC')
+            ->pluck('year')
+            ->toArray();
```

Comment:
not sure this is the best idea for getting a list of years, but I guess we'll find out...